components/ui/Avatar.tsx

```diff
@@ -16,15 +16,15 @@ const Avatar = observer(
     zoomed?: boolean;
     size?: number;
   }) => {
+    if (address === "") {
```

Comment:
Does this fix this https://github.com/zeitgeistpm/ui/issues/131? If so could you link it?